modules/update-best-practices.adoc

```diff
@@ -0,0 +1,57 @@
+
+While this design helps ensure that updates are successful as long as some key conditions are met, there are a number of actions you can take to increase your chances of a successful cluster update.
+
+[discrete]
+[id="recommended-versions_{context}"]
+=== Choose versions recommended by the OpenShift Update Service
+
+The OpenShift Update Service (OSUS) provides update recommendations based on cluster characteristics such as the cluster's subscribed channel, which are then saved by the Cluster Version Operator as either recommended or conditional updates.
+While it is possible to attempt an update to a version that is not recommended by OSUS, doing so significantly increases the risk of update failure or unintended consequences to the cluster after the update has finished.
+
+Choose only update targets that are recommended by OSUS to ensure a successful update.
+
+[discrete]
+[id="critical-alerts_{context}"]
+=== Address all critical alerts on the cluster
+
+Critical alerts must always be addressed as soon as possible, but it is especially important to address these alerts and resolve any problems before initiating a cluster update.
+Failing to address critical alerts before beginning an update can cause a loss of data or a major failure of cluster services.
+
+You should also periodically review Warning and Info alerts on the cluster to address any potentially problematic conditions before initiating an update.
+
+[discrete]
+[id="cluster-upgradeable_{context}"]
+=== Ensure that the cluster is in an Upgradable state
+
+When one or more Operators have not reported their `Upgradeable` condition as `true` for more than an hour, the `ClusterNotUpgradeable` warning alert is triggered in the cluster.
+In most cases patch updates are not blocked by this alert, but you cannot perform a minor version update until this alert is resolved and all Operators report `Upgradeable` as `true`.
+
+[discrete]
+[id="nodes-ready_{context}"]
+=== Ensure that all nodes are available
+
+// Completely guessing the explanation in this section just to have something to start with when this is reviewed by an SME.
+Nodes should not be down when beginning an update.
+Nodes that are not running and available may limit a cluster's ability to perform an update with minimal disruption to cluster workloads.
+
+Depending the on the configured value of the cluster's `maxUnavailable` spec, an unavailable node can also prevent itself and other nodes from having machine configuration changes applied during a cluster update.
+
+[discrete]
+[id="pod-disruption-budget_{context}"]
+=== Ensure that the cluster's PodDisruptionBudget is properly configured
+
+The `PodDisruptionBudget` object allows you to define the minimum number or percentage of pod replicas that must be available at any given time.
+This configuration allows workloads to be protected from disruptions during maintenance tasks such as cluster updates.
```

Comment:
I think we can afford to be a bit more specific:

1. Make sure that for HA workloads, there are replicas that can be temporarily taken offline without PDB configuration prohibiting that
2. Make  sure that non-HA workloads are either not protected by PDBs, or that there is an alternative reliable mechanism that allows draining these workloads eventually (such as periodic restart)